ko/lessons/basics/strings.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: page
+title: 문자열
+category: basics
+order: 14
+lang: ko
+---
+
+Elixir에서의 문자열, 문자 리스트, 문자소 그리고 코드 포인트에 대해 다뤄 보겠습니다.
+
+{% include toc.html %}
+
+## Elixir에서의 문자열
+
+Elixir에서 문자열은 바이트 시퀀스에 불과하지 않습니다. 다음의 예시를 봅시다:
```

Comment:
Oh, thank you. 
But I think using `리스트` instead of `시퀀스` can be confusing.
In this lesson, this document explain about `char list(문자 리스트)` and previous lesson also explains `list comprehension(리스트 해석)`. So, I think translating sequence into `리스트` is something awkward.